test/Source/DoctrineORMQuerySourceIteratorTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Exporter\Test\Source;
+
```

Comment:
Wrong namespace

Reply:
What should be the real namespace?

Reply:
What do you mean @core23 ? this is the namespace for the other test classes on this folder.

If it is wrong, maybe we should change it on a separate PR.

Reply:
Is this class used as an internal test (namespace: `tests`) or could we use this in other projects too (namespace: `tests`)?